StRoot/StBFChain/BigFullChain.h

```diff
@@ -1795,6 +1795,7 @@ Bfc_st BFC[] = { // standard chains
 #endif
   {"dEdxY2"       ,"dEdxY2","","tpcDb,StEvent","StdEdxY2Maker","libMinuit,StdEdxY2Maker"
    ,                                                                 "Bichsel method used for dEdx",kFALSE},
+  {"CalcdNdx", "", "", "", "StdEdxY2Maker", "",       "Option for StdEdxY2Maker to calculate dN/dx",kFALSE},
```

Comment:
Probably better not to have this option require the StdEdxY2Maker maker, but rather the dEdxY2 option:
"CalcdNdx", "", "", "dEdxY2", "","",...

Reply:
Done

Reply:
Almost.... Dependent options go in the 4th element of the array, not the 5th.